roles/uptime_kuma/tasks/main.yml

```diff
@@ -23,5 +23,13 @@
 - name: Create directories
   ansible.builtin.include_tasks: "{{ resources_tasks_path }}/directories/create_directories.yml"
 
+- name: "Migrator | Reset ownership of 'opt' directory for this app"
```

Comment:
The directory creation resource called above takes care of this. If it needs to recurse there is an additional var we can define, but it’s not recursing in your migrator.

Reply:
@owine does it take care at each re-run of roles ? or just at initial dir creation ?

Reply:
It works as said, thanks ! :)